phoenix-core/src/main/java/org/apache/phoenix/hbase/index/table/CoprocessorHTableFactory.java

```diff
@@ -42,14 +38,9 @@ public CoprocessorHTableFactory(CoprocessorEnvironment e) {
 
     @Override
     public HTableInterface getTable(ImmutableBytesPtr tablename) throws IOException {
-        Configuration conf = e.getConfiguration();
-
-        // make sure we use the index priority writer for our rpcs
-        IndexQosCompat.setPhoenixIndexRpcController(conf);
 
         // make sure we include the index table in the tables we need to track
         String tableName = Bytes.toString(tablename.copyBytesIfNecessary());
```

Comment:
Declare tableName as byte[] instead. I don't see any value in that debug log line, but if you do I guess you can leave it. Otherwise, just ditch it.